lidar:
  topic: /points_raw
  T_lidar_to_base:
    [
      [1.0, 0.0, 0.0, 0.0],
      [0.0, 1.0, 0.0, 0.0],
      [0.0, 0.0, 1.0, 0.68795841],
      [0.0, 0.0, 0.0, 1.0],
    ]
  T_lidar_to_imu:
    [
      [0.994486, -0.052957, 0.090512, 0.08704262],
      [0.054145, 0.998476, -0.010715, -0.11147986],
      [-0.089807, 0.015557, 0.99583884, 0.60000775],
      [0.0, 0.0, 0.0, 1.0],
    ]
  time_offset_lidar_to_imu: 0.00652
  fov_degree: 360
  blind: 0.5
  scan_line: 16
  scan_rate: 10
